source/NPC.h

```diff
@@ -61,7 +61,8 @@ class NPC {
 		CAPTURE,
 		DESTROY,
 		// Can be triggered by either the CAPTURE or DESTROY events.
-		KILL
+		KILL,
+		ENCOUNTER,
```

Comment:
comment pls, new category of triggers this is?

Reply:
Oh—it looks like it might actually want to be moved above `KILL`, since it is, in fact, corresponding to a `ShipEvent`.

Is that a safe thing to do—it won't mess with anything to change the underlying values of existing enum members?

Reply:
The raw value of these enums doesn't actually matter. It does for ShipEvent, though.

Reply:
Gotcha. Thanks for setting them up as suggestions!